### PR TITLE
built taxonomies

### PR DESCRIPTION
**Pull request by stephanegigandet**

### CODE REVIEW

File: taxonomies/additives.result.txt [hunk 1]
```diff
@@ -27462,22 +27554,22 @@ vegan:en: no
...
 cs:E543, E543 food additive
 da:E543, E543 food additive
 de:E543, E543 food additive
 el:E543, E543 food additive
 es:E543, E543 food additive
 et:E543, E543 food additive
-fi:E543, Kalsiumnatriumpolyfosfaatti, Kalsiumnatriumpolyfosfaattia
-fr:E543, Polyphosphate de sodium de calcium
+fi:E543, E543 food additive
+fr:E543, E543 food additive
```
Comment (teolemon) [inline taxonomies/additives.result.txt:27566]: No ?

File: taxonomies/additives.result.txt [hunk 2]
```diff
@@ -27462,22 +27554,22 @@ vegan:en: no
...
 hu:E543, E543 food additive
 it:E543, E543 food additive
 lt:E543, E543 food additive
 lv:E543, E543 food additive
 mt:E543, E543 food additive
-nl:E543, Natriumcalciumpolyfosfaat
+nl:E543, E543 food additive
```
Comment (teolemon) [inline taxonomies/additives.result.txt:27572]: is that a desired change ?

**Review by teolemon — APPROVED** [review]
I'm approving, but all those changes are overwhelming.

File: taxonomies/additives.result.txt (hunk 2 above)
Comment (stephanegigandet) [reply taxonomies/additives.result.txt:27572]: yes. see @aleene 's comment:

#comment:en:The same as E452iii, put any synonyms there
en:E543, E543 food additive

**Pull request merged**